static/context.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react'
+import { AsyncStorage } from 'react-native'
+import FormData from 'FormData'
+
+import api from './api'
+import { content } from './content'
+
+export const AppContext = React.createContext()
+export const AppConsumer = AppContext.Consumer
+
+export class AppProvider extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      userId: ''
+    }
+  }
+
+  async login (email, password) {
```

Comment:
I'm not sure how `Context` works.
It feels misplaced to have the `login()` function here intead of holding it inside the `AuthScreen`.
It feels like leaving the _component_ approach as this depends on the AuthScreen and the other way around.

Can't the AuthScreen send the user infos to the _Context_?

Reply:
yes sure and now I think it's a better idea, I was just trying ... and that was not working so ...